backends/magma/ceed-magma-basis.c

```diff
@@ -8,6 +8,15 @@
 #include <ceed/ceed.h>
 #include <ceed/backend.h>
 #include "ceed-magma.h"
+#include <ceed/jit-tools.h>
+#include "ceed-magma.h"
+#ifdef HAVE_HIP
```

Comment:
Where is this macro definition coming from? It looks a bit strange not having some `MAGMA_` or `CEED_` leading it.

Reply:
Oh yeah, back when I originally added the HIP option for the MAGMA backend, `HAVE_HIP` was the macro used in MAGMA itself, and it had to be defined for some header definitions or something like that (I don't remember the exact details).   But MAGMA itself has actually transitioned to `MAGMA_HAVE_HIP`, yet it doesn't cause errors with compilation anymore if they don't match...
Anyway, we should probably change to `MAGMA_HAVE_HIP` here, too.  I can make that change.

Reply:
Or actually, if the macro doesn't need to exactly match MAGMA anymore for header inclusion purposes, would it be preferred if we used a different one here?  like `CEED_MAGMA_USE_HIP` or something?

Reply:
I like `CEED_BACKEND_SPECIFIC_MACRO` like `CEED_MAGMA_USE_HIP`

Reply:
Done in acc154bdaa78f7d5eb152bc74e55892088677db5